src/test/ui/single-use-lifetime/one-use-in-trait-method-argument.stderr

```diff
@@ -14,7 +14,7 @@ LL | #![deny(single_use_lifetimes)]
 help: elide the single-use lifetime
    |
 LL |     fn next(&mut self) -> Option<Self::Item> {
-   |           ----
+   |           /\/\
```

Comment:
This case seems better than the current. The current is hard to notice that we remove two `'g`s, yeah.